variable/energy/energy-secondary.yaml

```
@@ -41,18 +41,64 @@ Secondary Energy|Electricity|Fossil:
    unit: EJ/yr
 
 Secondary Energy|Electricity|Gas:
-   description: Net electricity production from natural gas
+   description: Net electricity production from methane
```

I see why this may be "more correct", but I think it is detrimental to 
```suggestion
   description: Net electricity production from natural gas (including methane from biomass or hydrogenation)
```